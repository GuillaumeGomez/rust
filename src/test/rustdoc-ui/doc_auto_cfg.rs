// check-pass
#![feature(doc_auto_cfg)]
