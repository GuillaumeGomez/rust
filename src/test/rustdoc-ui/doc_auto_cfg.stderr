warning: feature `doc_auto_cfg` now requires attribute `#![doc(auto_cfg)]` to enable the behavior

warning: 1 warning emitted

